<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reservation</class>
 <widget class="QMainWindow" name="Reservation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>851</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Reserver</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>nom Client</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="client"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>N Chambre</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="num"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>date Debut</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="dateD"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Date Fin</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="dateF"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Nombre jours</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="njour"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Reserver</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Afficher Reservation</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>0</y>
        <width>721</width>
        <height>241</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>id Reservation</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>id Client</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>id Chambre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>date Debut</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date Fin</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>nombre jours</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>montant totale</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Mise Ã  jour</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>supprimer</string>
     </attribute>
     <widget class="QLabel" name="message">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>411</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>id Reservation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="messagesupp"/>
       </item>
       <item>
        <widget class="QPushButton" name="supprimerButton">
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Facteur</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>411</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>CNIE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="messagesupp_2"/>
       </item>
       <item>
        <widget class="QPushButton" name="supprimerButton_2">
         <property name="text">
          <string>Afficher Facteur</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="erreur">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>511</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>240</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Mantont</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="montantF">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>291</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>nom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nomF">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>prenom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="prenomF">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>CNIE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cnieF">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>N telephone</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="telephoneF">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Nombre des chambres</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="chambreF">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Page d'acueille</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
